run_worker.sh

```diff
@@ -34,17 +33,12 @@ docker run \
     -e WT_VOLUMES_PATH="$PWD"/volumes \
     -e MATLAB_FILE_INSTALLATION_KEY=${matlab_file_installation_key} \
     -v /var/run/docker.sock:/var/run/docker.sock:ro \
-    -v /:/host \
-    -v /var/cache/davfs2:/var/cache/davfs2 \
-    -v /run/mount.davfs:/run/mount.davfs \
+    -v /tmp:/tmp \
     -v $PWD/src/gwvolman:/gwvolman \
     -v $PWD/src/girderfs:/girderfs \
+    --mount type=bind,source="$PWD/volumes",target="$PWD/volumes",bind-propagation=rshared \
```

Comment:
Just noting that this works on deploy-dev because licenses are stored under volumes, which isn't the case on production. We can either move the prod licenses under /mnt/homes (which is probably more convenient) or add a second bind mount.